File: ClientApp/src/app/Models/reservation-day.model.ts
export class ReservationDayModel {
  public id: string;
  public username: string;
  public userID: string;
  public isReserved: boolean;
  public isPending: boolean;
  public dateOfReservation: string

  constructor(id: string, username: string, userID: string, isReserved: boolean, isPending: boolean, dateOfReservation: string) {
    this.id = id;
    this.username = username;
    this.userID = userID;
    this.isReserved = isReserved;
    this.isPending = isPending;
    this.dateOfReservation = dateOfReservation;
  }
}

File: Models/ReservationDayModel.cs
namespace MirekInzFinal.Models
{
    public class ReservationDayModel
    {
        public string ID { get; set; } = string.Empty;

        public string Username { get; set; }= string.Empty;

        public string UserID { get; set; } = string.Empty;

        public bool IsReserved { get; set; }

        public bool IsPending { get; set; }

        public DateTime DateOfReservation { get; set; }

        public ReservationDayModel(string iD, bool isReserved, DateTime dateOfReservation)
        {
            ID = iD;
            IsReserved = isReserved;
            DateOfReservation = dateOfReservation;
        }
    }
}